picoquic/tls_api.c

```diff
@@ -1362,7 +1362,8 @@ int picoquic_tlsinput_stream_zero(picoquic_cnx_t* cnx)
         }
         ret = 0;
     } else {
-        ret = picoquic_connection_error(cnx, PICOQUIC_TLS_HANDSHAKE_FAILED);
+        (void)picoquic_connection_error(cnx, PICOQUIC_TLS_HANDSHAKE_FAILED);
```

Comment:
Hmm this line makes on of my tests in `picoquic-rs` flakly. Sometimes the handshake error is does not reach the other side.